examples/content-type-selection-config/README.md

```diff
@@ -0,0 +1,18 @@
+# Assigning an app to ContentTypes
+
+We recommend using `TargetState` to assign apps to specific ContentTypes in your app's
```

Comment:
"the `targetState` datastructure" or "the `targetState` pattern" or simply "Target State".
Should be consistent with naming here: https://www.contentful.com/developers/docs/extensibility/app-framework/target-state/

Reply:
Also, it should be "content type" as in the docs